_common/KeymanSentry.php

```diff
@@ -5,9 +5,11 @@
 
   class KeymanSentry {
     static function init($dsn) {
-      if(isset($_SERVER['SERVER_NAME'])) {
+      $env = getenv();
+
+      if(isset($env['SERVER_NAME'])) {
```

Comment:
`SERVER_NAME` is a standard `$_SERVER` property: https://www.php.net/manual/en/reserved.variables.server.php

```suggestion
      if(isset($_SERVER['SERVER_NAME'])) {
```

Reply:
gotcha - SERVER_NAME will always be set.